creme/creme_core/templates/creme_core/detail/entity-filter.html

```diff
@@ -17,24 +18,12 @@
   {% with edition_url=object.get_edit_absolute_url deletion_url=object.get_delete_absolute_url %}
    {% if edition_url %}
     <div class='bar-action'>
-        <a href="{{edition_url}}" class="bar-button{% if not edition_perm %} forbidden is-disabled{% endif %}">
-            {% widget_icon name='edit' size='brick-hat-bar-button' label=_('Edit') %}
-        </a>
+      {% brick_bar_button action='redirect' label=_('Edit') url=edition_url icon='edit' enabled=edition_perm %}
     </div>
    {% endif %}
    {% if deletion_url %}
     <div class='bar-action'>
-{# NO A BRICK  {% brick_bar_button action='creme_core-detailview-delete' label=_('Delete') url='creme_core__delete_efilter'|url icon='delete' confirm=_('Do you really want to delete this filter?') enabled=deletion_perm %} #}
-      {% if deletion_perm %}{% translate 'Do you really want to delete this filter?' as confirm_msg %}
-        <a onclick="creme.utils.confirmSubmit(this, '{{confirm_msg|escapejs}}');" class="bar-button">
-            {% widget_icon name='delete' size='brick-hat-bar-button' %}
-            <form method="POST" action="{{deletion_url}}" style="display:none;">{% csrf_token %}</form>
-        </a>
-      {% else %}
-        <span class="bar-button forbidden is-disabled">
-            {% widget_icon name='delete' size='brick-hat-bar-button' %}
-        </span>
-      {% endif %}
+      {% brick_bar_button action='creme_core-hatmenubar-delete' label=_('Delete') url=deletion_url icon='delete' confirm=_('Do you really want to delete this filter?') enabled=deletion_perm __redirect='creme_config__efilters'|url %}
```

Comment:
As indicated in my comment, this hat bar is bot a real brick ; it has not brick-id (& dependencies etc...) & cannot be reloaded. I presume your new code works fine because it performs a redirect, so reloading is not an issue.
Maybe we should add a comment about this, because it could be confusing, no?

Reply:
I agree it will be confusing.